lesson/lib/prefect_workflows.py

```diff
@@ -0,0 +1,209 @@
+import os
```

Comment:
Quite nice to see the functions in the file. Just a few things that I think could improve the reading flow: 

- Separate the tasks from the flows under sections marked by comments like: 
`#####  Tasks  #####`
- Use more expressive naming (e.g. `taxi_rides` instead of `df`) -> I have implemented this [here](https://github.com/artefactory/xhec-mlops-crashcourse/pull/13)
- Convert docstrings to google format